core/src/main/java/tc/oc/pgm/tablist/PlayerOrder.java

```diff
@@ -61,6 +61,6 @@ public int compare(MatchPlayer ma, MatchPlayer mb) {
     }
 
     // All else equal, order the players alphabetically
-    return a.getName(viewer).compareToIgnoreCase(b.getName(viewer));
+    return a.getName().compareToIgnoreCase(b.getName());
```

Comment:
Is this going to cause problems?

Reply:
This will cause some strange ordering if another plugin is using the SportPaper fake name api